scripts/publish-docker.sh

```diff
@@ -13,12 +13,18 @@ base_tag="$2"
 hash="$3"
 version="$4"
 artifacts="$5"
+ngc_registry="nvcr.io/isv-ngc-partner/determined"
 
 underscore_name="$(echo -n "$log_name" | tr - _)"
 
 docker push "$base_tag-$hash"
 docker push "$base_tag-$version"
 
+docker tag "$base_tag-$hash" "$ngc_registry/$base_tag-$hash"
+docker tag "$base_tag-$version" "$ngc_registry/$base_tag-$version"
+docker push "$ngc_registry/$base_tag-$hash"
+docker push "$ngc_registry/$base_tag-$version"
```

Comment:
What's the plan for these images?  If you run our software in NGC, will you get them automatically?  Where will we encode that logic?

Reply:
The plan is to just push the images to the NGC repo. Users can download images from the NGC registry. That's all I need to do. They need to manually specify the image name though.

Reply:
Ok.  Would it be possible to put it as the default task environment in the nvidia helm chart?  (Is there an ngc-related helm chart?)

Reply:
Actually, I didn't make a separate helm chart for nvidia. It shares the same helm chart. They just need to override the registry path in the `values.yaml` for NGC images.